include/SSystem/SComponent/c_bg_s_gnd_chk.h

```diff
@@ -18,7 +18,7 @@ class cBgS_GndChk : public cBgS_Chk, public cBgS_PolyInfo {
 
     f32 GetNowY() const { return mNowY; }
     void SetNowY(f32 y) { mNowY = y; }
-    const cXyz& GetPointP() { return m_pos; }
+    cXyz& GetPointP() { return m_pos; }
```

Comment:
from what i understand currently, getter functions that end with p or P are for getting pointers, while otherwise i think its references.